clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/GvcfTests.scala

```diff
@@ -570,10 +572,10 @@ trait GvcfTests { self: BaseIntegrationSpec =>
     val detailMetricsContents =
       s"$randomId --- I am dummy detail metrics --- $randomId"
 
-    val gvcfName = s"$randomId.gvcf"
-    val indexName = s"$randomId.index"
-    val summaryMetricsName = s"$randomId.summary.metrics"
-    val detailMetricsName = s"$randomId.detail.metrics"
+    val gvcfName = s"$randomId.g.vcf.gz"
+    val indexName = s"$randomId.g.vcf.gz.tbi"
+    val summaryMetricsName = s"$randomId.variant_calling_summary_metrics"
+    val detailMetricsName = s"$randomId.variant_calling_detail_metrics"
```

Comment:
Even though this is test code, I think we want apply the same rigor to it regarding code duplication. Can the common test setup code be refactored into a common method?

Reply:
Done! Glorious -642 lines

Reply:
![bitmoji](https://render.bitstrips.com/v2/cpanel/3c97455d-0ce1-4ac8-82a0-601e6d6f6fd2-58bd1e74-5b74-4b76-b7ec-5ecb58ede85e-v1.png?transparent=1&palette=1&width=246)